hack/generate-buildah-remote.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+DIR=`dirname $0`
+docker run -v $DIR/..:/data quay.io/redhat-user-workloads/rhtap-build-tenant/multi-arch-controller/multi-arch-controller:taskgen-221979523603b730feca730aeb9d43f24a3c1e67  --buildah-task=/data/task/buildah/0.1/buildah.yaml --remote-task=/data/task/buildah-remote/0.1/buildah-remote.yaml
```

Comment:
Need `:Z` here?

Reply:
Why?

Reply:
For SELinux: https://docs.podman.io/en/latest/markdown/podman-run.1.html#volume-v-source-volume-host-dir-container-dir-options
(see "Labeling systems like SELinux ..."

It does work for me without the :Z though, odd. Usually I need to add it for volume mounts

Reply:
oh, odd. It didn't work for me without Z even for read-only operations in the past, so I always add it for volume mounts.